### ISSUE TITLE
Offset for the rest of the charts.

### ISSUE THREAD

**Issue by jwerre** [andreaferretti/paths-js#54]
I noticed there's an offset for Bar but not for the rest of the charts. It would be nice to have an offset for Pie, Smooth Line, etc as well.

**Comment by andreaferretti** [comment]
You are right. For now, let me offer two suggestions:

* For Pie and Radar charts, there is no need to have an `offset` parameter, since there is `center` which determines where the graph goes anyway
* In general, you can do without the `offset` parameter by putting something like `transform="translate(20,30)"` as an attribute of the SVG element

**Comment by jwerre** [comment]
Yes, I saw the center for Pie and Radar—perfect! Unfortunately, I’m drawing charts on the server side and rendering them to pdf so no SVG element  .
